README.md

```diff
@@ -1,45 +1,96 @@
 ## Faros CI/CD Github Action
 
-A GitHub Action to emit CI/CD information for builds and deployments from a GitHub workflow
+A GitHub Action to report CI/CD information for builds and deployments from a GitHub workflow
 context variables to Faros API.
 
 ## Usage
 
+See [action.yml](action.yml) for the full documentation for this action's inputs and outputs.
+
+### Report a Build Event To Faros CI/CD Build Model
+
+To report a build event to Faros specify build in the `model` parameter and the build details.
+
 ```yaml
 - name: Emit build info to Faros
   uses: faros-ai/faros-cicd-github-action@v1
   with:
     api-key: ${{ secrets.FAROS_API_KEY }}
+    api-url: ${{ env.FAROS_API_URL }}
     model: build
-    status: success
+    build-pipeline-id: build-deploy-workflow
+    status: Success
     started-at: 1594938057000
-    ended-at: 1594948057000
-    build-pipeline-id: faros-cicd-github-action-build-deploy
+    ended-at: 1594948069000
 ```
 
-## Usage
+> :clipboard: Note: Whilst the build-pipeline-id is optional, it is recommended to provide one to uniquely identify this workflow in Faros since multiple GitHub workflows can have the same name. If the build-pipeline-id is not provided it will be generated from the workflow name in the lowercase format GITHUB_ORG/REPO/GITHUB_WORKFLOW_NAME.
+
+### Report a Deployment Event To Faros CI/CD Deployment Model
+
+To report a build event to Faros specify `deploy` in the `model` parameter and the deployment details. To ensure the build is correctly linked to the build, provide the build model keys, i.e. `build-id, build-pipeline-id, build-org-id, build-platform`.
 
 ```yaml
-- name: Emit build info to Faros
+- name: Emit deployment info to Faros
```

Comment:
why are there two deployment events in the example? and why the first does not have a status?

Reply:
First should have a status. For the 2nd event, thought it would be useful to show usage of using the steps from the previous job. Can remove and just maybe add an example of how to reference one output

Reply:
I think it's enough to have one build event and one deployment event example

Reply:
and with comments:
```
- name: Emit build info to Faros
  uses: faros-ai/faros-cicd-github-action@v1
  with:
    api-key: ${{ secrets.FAROS_API_KEY }}
    api-url: ${{ env.FAROS_API_URL }}
    model: build
    build-pipeline-id: build-deploy-workflow # explain what is this for
    status: Success                          # show some other possible valiues
    started-at: 1594938057000
    ended-at: 1594948069000
```